clients/p4rt-ctl/p4rt-ctl.in

```diff
@@ -74,6 +74,7 @@ for P4Runtime switches:
   get-action-profile-group SWITCH PROFILE FLOW  prints group entries
 
   get-counter SWITCH CTR_TBL CTR_FLOW   gets indirect counter table entry
+  get-direct-counter SWITCH CTR_TBL CTR_FLOW   gets direct counter table entry
```

Comment:
Do we need a set/reset counter in addition to get-counter? What is the requirement here?

Reply:
the requirement was to read the direct counter stats for the traffic wrt configured rule in FxP